test/Emitter/SapiStreamEmitterTest.php

```diff
@@ -53,7 +56,7 @@ public function testEmitCallbackStreamResponse(): void
             ->withBody($stream);
         ob_start();
         $this->emitter->emit($response);
-        $this->assertSame('it works', ob_get_clean());
+        self::assertSame('it works', ob_get_clean());
```

Comment:
Oh, I hadn't seen this one 😁

PHPUnit has an assertion to verify the output of the execution. It's something that we should address in another PR, though.